src/hotspot/share/opto/compile.cpp

```diff
@@ -622,6 +622,7 @@ Compile::Compile( ciEnv* ci_env, ciMethod* target, int osr_bci,
                   _has_reserved_stack_access(target->has_reserved_stack_access()),
 #ifndef PRODUCT
                   _igv_idx(0),
+                  _igv_phase_iter(),
```

Comment:
This is value initialization which guarantees proper zeroing, right? For other arrays, for example `Compile::_trap_hist`, we use explicit `Copy::zero_to_bytes` but I think your variant is fine.

Reply:
Yes, correct. I'll switch to the `Copy::zero_to_bytes` initialization, better to be consistent.